src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.test.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import times from 'lodash/times';
+
+import { ProteinExternalReferenceGroup } from './ProteinsListItemInfo';
+import { createExternalReference } from 'tests/fixtures/entity-viewer/product';
+import { ExternalSource } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
+
+jest.mock(
+  'src/shared/components/instant-download/instant-download-protein/InstantDownloadProtein',
+  () => jest.fn()
+);
+
+const swissprotXref = createExternalReference({
+  accession_id: 'SWISS123',
+  source: {
+    name: 'UniProtKB/Swiss-Prot',
+    id: 'swiss_123'
+  }
+});
+
+const tremblSource = {
+  name: 'UniProtKB/TrEMBL',
+  id: 'trembl1'
+};
+const tremblXrefs = times(4, () =>
+  createExternalReference({ source: tremblSource })
+);
+
+times(4, (index) => (tremblXrefs[index].accession_id = `trembl_${index}`));
+
+describe('<ProteinsListItemInfo /', () => {
+  it('renders a single xref', () => {
+    const props = {
+      source: ExternalSource.UNIPROT_SWISSPROT,
+      xrefs: [swissprotXref]
+    };
+    // ProteinExternalReferenceGroup expect xrefs from a single source
+    const { queryByText } = render(
+      <ProteinExternalReferenceGroup {...props} />
+    );
+    expect(queryByText(swissprotXref.source.name)).toBeTruthy();
+    expect(queryByText(swissprotXref.accession_id)).toBeTruthy();
+  });
+
+  describe('multiple xrefs', () => {
+    it('renders 2 xrefs', () => {
```

Comment:
i suppose we'd better change this test to check for 3 xrefs to prove to ourselves that all three get rendered, and that there is no chevron.